app/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM ubuntu:20.04 AS base
```

Comment:
minor: you could consider using the latest LTS ubuntu, this one is already an older version.

Reply:
Ok. I was discussing with Pedro about having a more standardized set of base images, so the building might be a bit faster. 
I'd be happy to adopt whatever we decide (apart from special cases, where using a more specialized image might be an advantage).